run_dir/static/js/projects.js

```diff
@@ -10,9 +10,36 @@ var projects_page_type = $('#projects-js').attr('data-projects');
 var begin = new Date(2012,0, 1); 
 var end = new Date();
 
+$('#projectFilterDate').on('shown.bs.modal', function () {
+ $("#OpenDateSlider").dateRangeSlider('resize');
+ $("#QueueDateSlider").dateRangeSlider('resize');
+ $("#CloseDateSlider").dateRangeSlider('resize');
+})
 $(document).ready(function() {
-    //load the slider
-    $("#dateSlider").dateRangeSlider({
+    //load the sliders
+    $("#OpenDateSlider").dateRangeSlider({
```

Comment:
Can we use a HTML class here instead of an ID and have one copy of this code instead of three?